src/components/Dropdown/Dropdown.tsx

```diff
@@ -51,15 +51,25 @@ const Dropdown: React.FunctionComponent<DropdownProps> = ({
         aria-label={ariaLabel}
         onChange={checkHandleOnChange}
       >
-        {placeholder && <option value="">{placeholder}</option>}
-        {options.map(({ label, value, disabled, selected }) => {
+        {placeholder && (
+          <option
+            className={classes.option}
+            value={placeholder.value}
+            selected={placeholder.value === defaultValue}
+            disabled={placeholder.disabled}
+          >
+            {placeholder.label}
+          </option>
+        )}
+
+        {options.map(({ label, value, disabled }) => {
           return (
             <option
               key={label}
               value={value}
               className={classes.option}
               disabled={disabled}
-              selected={selected}
+              selected={value === defaultValue}
```

Comment:
:+1: Now that you have `options` as an array of `Options` and `placeholder` as an optional `Option` you can clean up the code even further.